TwitchChannelPointsMiner/classes/TwitchChat.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from multiprocessing import Process
+
+from TwitchChannelPointsMiner.classes.entities.Chat import Chat
+
+logger = logging.getLogger(__name__)
+
+
+class ChatSettings(object):
+    def __init__(
+        self,
+        username: str = None,
+        token: str = None,
+    ):
+        self.username = username
+        self.token = token
+
+
+class TwitchChat(Process):
```

Comment:
It's a long running-job with lower I/O.
We should extend a `Thread` instead of `Process`